src/shell/dotnet/Shell/Shell.csproj

```diff
@@ -46,4 +47,8 @@
 		<ProjectReference Include="..\..\..\messaging\dotnet\src\Host\MorganStanley.ComposeUI.Messaging.Host.csproj" />
 	</ItemGroup>
 
+	<Target Name="PreBuild" BeforeTargets="PreBuildEvent">
+	  <Exec Command="PowerShell cd ..\..\..\..\ ; .\src\shell\dotnet\copy-fdc3-iife-bundle.ps1" />
```

Comment:
`powershell cd $(ComposeUIRepositoryRoot); .\src\shell\dotnet\copy-fdc3-iife-bundle.ps1`

But it would be better if the script didn't require the repo root as working directory.